setup.py

```diff
@@ -0,0 +1,27 @@
+from setuptools import setup
+
+setup(
+    name='MoleculeVAE-re-implementation',
+    version='1.0.0',
+    description='MolecularVAE',
+    packages=['MoleculeVAE-reImplementation'],
+    url='https://github.com/timcanby/MoleculeVAE-re-implementation',
+    license='',
+    author='kangyingli',
+    author_email='kangyinglee0501@gmail.com',
```

Comment:
I don't think you have to put an e mail if you don't want to.
I don't like having my personal e mail online all over the place :)